src/main/steps/c100-rebuild/safety-concerns/abduction/passport-office-notified/template.njk

```diff
@@ -1,11 +1,8 @@
 {% extends "common/template.njk" %}
-
```

Comment:
@niranjan-301 Please try to avoid these changes in other files which you have not touched or worked on.